style/style.css

```diff
@@ -0,0 +1,154 @@
+/* Typography */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+#container {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #dbdada;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  padding-bottom: 30px;
+}
+
+.title {
+  font-size: 45px;
+  margin-top: 50px;
+  font-weight: 700;
+  text-align: center;
+  color: #8a2be2;
+}
+
+cite {
+  color: black;
+}
+
+.image {
+  width: 80%;
+  margin-top: 40px;
+  margin-bottom: 25px;
+  max-width: 900px;
+}
+
+.img-caption {
+  font-size: 17px;
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+span {
+  font-size: 2.1em;
+  line-height: 3;
+  text-align: center;
+}
+
+.img-div {
+  width: 90%;
+  height: auto;
+  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  margin-bottom: 40px;
+  flex-direction: column;
+  border-radius: 10px;
+  background-color: rgb(214, 212, 212);
+  box-shadow: 0 0 30px rgba(125, 128, 126, 0.993);
+}
+
+.tribute-info {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.tribute-info ul {
+  width: 60%;
+}
+
+.tribute-info ul li {
+  margin: 30px 0;
+}
+
+.main p {
+  font-size: 20px;
+  width: 90%;
+  text-align: center;
+}
```

Comment:
Great work for eliminating multiple ID selectors that were making for code mess👏👏👏